Sources/ContextPropagation/Instrumentation/InstrumentationMiddleware.swift

```diff
@@ -18,6 +18,10 @@ public struct InstrumentationMiddleware<InjectInto, ExtractFrom>: Instrumentatio
         self.inject = inject
     }
 
+    public init<M: InstrumentationMiddlewareProtocol>(_ middleware: M) where M.InjectInto == InjectInto, M.ExtractFrom == ExtractFrom {
```

Comment:
That's a super long line, can we cut it down like so perhaps:

```suggestion
public init<Middleware>(_ middleware: Middleware)
        where
        Middleware: InstrumentationMiddlewareProtocol,
        Middleware.ExtractFrom == Self.ExtractFrom,
        Middleware.InjectInto == Self.InjectInto {
```